test/test.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs');
+const parser = require('./../src/parser.js');
+const chai = require('chai');
+const docDefault = fs.readFileSync('./test/config/doc.txt', 'utf8');
+const configDefault = require('./../src/template/configDefault.json');
+const mocha = require('mocha');
+const assert = require('assert');
```

Comment:
This block needs to divide into parts, e.g.: `Import defaults settings`, `Import core functions` and e.t.c

Reply:
@svoboda-rabstvo yes, but JSDoc adding is another task.